src/ripple/test/impl/WSClient.cpp

```diff
@@ -201,6 +201,115 @@ class WSClientImpl : public WSClient
         return {};
     }
 
+    Json::Value
+    invoke(Json::Value const& cmd) override
```

Comment:
Is there a plan to switch to one of the `invoke` forms?  If not, is it possible to have the old version invoke the new one to reduce code duplication?